puppet/zulip/manifests/localhost_camo.pp

```diff
@@ -1,14 +1,17 @@
 class zulip::localhost_camo {
-  include zulip::camo
+  if $::osfamily == 'debian' {
+    include zulip::camo
 
-  # Install nginx configuration to run camo locally
-  file { '/etc/nginx/zulip-include/app.d/camo.conf':
-    ensure  => file,
-    require => Package['nginx-full'],
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644',
-    notify  => Service['nginx'],
-    source  => 'puppet:///modules/zulip/nginx/zulip-include-app.d/camo.conf',
+    # Install nginx configuration to run camo locally
+    file { '/etc/nginx/zulip-include/app.d/camo.conf':
+      ensure  => file,
+      require => Package['nginx-full'],
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0644',
+      notify  => Service['nginx'],
+      source  => 'puppet:///modules/zulip/nginx/zulip-include-app.d/camo.conf',
+    }
   }
+  # Do not setup camo on systems other than Debian
```

Comment:
I think we want to do this at the `voyager.pp` layer -- i.e. not including `localhost_camo.pp`.

Reply:
Fixed.